.github/workflows/linter.yml

```
@@ -0,0 +1,62 @@
+name: Linter
+on: pull_request
+jobs:
+  Bandit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Run checks
+        run: |
+          pip install --user -r <(grep "^bandit" ./requirements.txt)
+          echo "Bandit version: "`bandit --version | head -1`
+          bandit -r ./
+  isort:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Run checks
+        run: |
+          pip install --user -r <(grep "^isort" ./requirements.txt)
+          isort --check --diff .
+  Pylint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Run checks
+        run: |
+          pip install --user -r <(grep "^pylint" ./requirements.txt)
+          echo "Pylint version: "`pylint --version | head -1`
+          git ls-files -z '*.py' | xargs -0 pylint -r n
```

```suggestion
          git ls-files -z '*.py' | xargs -0 pylint -r n -j 0
```
How about running it in parallel?

---

I agree in principle, but a) I think that goes beyond the scope of this PR (plus I don't want to mix meaningful changes with moving code around to avoid contaminating history), and b) it's still under a minute, so is it really that important? :-P